contrib/checkbox-ce-oem/checkbox-provider-ce-oem/bin/xtest_install_ta.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+
+from look_up_xtest import look_up_xtest
+from subprocess import run, CalledProcessError
+
+
+def run_command(cmd, capture_output=True, text=True, check=True):
+    try:
+        result = run(
+            cmd, capture_output=capture_output, text=text, check=check
+        )
+        return result.stdout.strip() if capture_output else None
+    except CalledProcessError as e:
+        raise SystemExit("Error: {}".format(e))
+
+
+def find_ta_path():
+    print("Looking for TA path...", flush=True)
+    cmd = ["find", "/var/snap", "-wholename", "*/lib/optee_armtz"]
+    path = run_command(cmd)
```

Comment:
we could use the Python built-in function `glob` function to lookup files instead of a subporcess command
you could refer to this scripts: https://github.com/canonical/checkbox/blob/main/providers/base/bin/wifi_client_test_netplan.py#L61

Reply:
pushed the fix